components/definitions/newTab.d.ts

```diff
@@ -173,6 +173,7 @@ declare namespace NewTab {
 
   export interface RewardsWidgetState {
     rewardsEnabled: boolean
+    rewardsUnsupportedRegion: boolean
```

Comment:
Maybe this should be `isUnsupportedRegion`?

Reply:
Fixed in 9f111ed50548b90b6c797a561ef0fec5f119a8ea